cmd/cortex/main.go

```diff
@@ -29,11 +30,21 @@ var (
 	Revision string
 )
 
+// configInfo exposes information about the loaded config
+var configInfo *prometheus.GaugeVec = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "cortex_config_hash",
+		Help: "Hash of the currently active config.",
```

Comment:
I would add "file" because config can also be specified via CLI flags.

```suggestion
		Help: "Hash of the currently active config file.",
```